src/util/KvAuth0.js

```diff
@@ -26,23 +30,20 @@ export default class KvAuth0 {
 		scope,
 		user = null,
 	}) {
+		this[errorCallbacks] = [];
 		this.enabled = true;
 		this.user = user;
 		this.accessToken = accessToken;
 		this.isServer = typeof window === 'undefined';
 
 		if (!this.isServer) {
-			// delay loading of webauth until we've confirmed we're in the browser,
-			// as it complains about not having window
```

Comment:
Guess this was fixed in some update?

Reply:
Yes, with version 1.4.1 of idtoken-verifier https://github.com/auth0/idtoken-verifier/commit/7e25214d2a610a1597939fa8d113778f8ae5d0d8